src/styles/brackets.less

```diff
@@ -486,11 +486,58 @@ a, img {
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
-        
+
+        &-text {
+            display: inline;
+        }
+
         .dark & {
             background-color: #1d1f21; // not using a variable on purpose.
             border-bottom-color: rgba(255, 255, 255, 0.05);
         }
+
+        &-flipview-btn {
+            position: relative;
+            display: inline-block;
+            top: 0px;
+            padding-top: 4px;
+            padding-right: 4px;
+            padding-bottom: 2px;
+            padding-left: 4px;
+            margin-left: 0;
+            margin-bottom: 0;
+            .sprite-icon(0, 0, 13px, 13px, "images/flip-view-icons.svg");
+            background-origin: content-box;
+            -webkit-transform: translateZ(0); // forces GPU mode for better filter rendering on retina
+            transform: translateZ(0); // future proofing
+            -webkit-filter: drop-shadow(0 1px 0 rgba(0,0,0,0.36));
```

Comment:
Thanks @le717, fixed those :+1: 

RE: the filter declaration, there are some other components using the plain vendor-prefixed versions of ones that have standardized ones available, might be worth checking out.